Course/IOS15(P1)/Lesson6/AccountView2.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  AccountView.swift
+//  Course
+//
+//  Created by Марк Горкій on 12.10.2022.
+//
+import SwiftUI
+import Foundation
+struct AccountView2: View {
+    @State var isDeleted = false
+    
+    var body: some View {
+        NavigationView {
+            List {
+                VStack(spacing: 8) {
+                    Image(systemName: "person.crop.circle.fill.badge.checkmark")
+                        .symbolVariant(.circle.fill)
+                        .font(.system(size: 32))
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(.blue, .blue.opacity(0.3))
+                        .padding()
+                        .background(Circle().fill(.ultraThinMaterial))
+                        .background(
+                            Image(systemName: "hexagon")
+                                .symbolVariant(.fill)
+                                .foregroundColor(.blue)
+                                .font(.system(size: 200))
+                                .offset(x: -50, y: -100)
+                    )
```

Comment:
This whole section can be stored in its own var x : some View to tidy up code and make it more readable. I know you're following tutorial and he will get to that anyway.